react/src/components/user/course/CourseChapters.jsx

```
@@ -0,0 +1,134 @@
+            </div>
+
+            <div className="sections-list">
+              {sections?.map((section) => (
+                <div key={section.id} className="section-card">
+                  <button 
+                    className="section-header"
+                    onClick={() => toggleSection(section.id)}
+                    aria-expanded={expandedSections[section.id]}
+                    aria-controls={`section-${section.id}`}
+                  >
+                    <div className="section-title">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)